extend/generic-extractor/configuration/api/authentication/query.md

```diff
@@ -48,3 +48,27 @@ However, we recommend using the `authentication` setting for credentials so that
 configuration does not become a complete mess.
 
 See [example [EX077]](https://github.com/keboola/generic-extractor/tree/master/doc/examples/077-query-auth).
+
+## Configuration With Cyphered Token Example
+Usually you want the value used for authentication cyphered (the `api-token` parameter with the value 2267709 in our example), so you are not exposing it to other users and also not storing it in the configuration versions history. The following authentication configuration combined with the parameter defined in the `config` section does that (the value with the prefix `#` is cyphered upon saving the configuration):
```

Comment:
prosím nalinkovat https://developers.keboola.com/extend/generic-extractor/configuration/config/
a nalinkovat EX094 jako analogii (případně vytvořit novej, pokud se ti to nezdá)

Reply:
Doplněno.